service/src/test/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Thu Jan 02 13:48:38 GMT 2025
```

Comment:
This file looks sus. Where did it come from?

Reply:
it was generated when I run the upgrade command: "./gradlew wrapper --gradle-version 8.8 --distribution-type bin"

Reply:
Why is it living inside of the `service/src/test` folder? We already have a gradle-wrapper.properties file within the expected `service/gradle/wrapper` folder.

Reply:
I can delete it. I think Gradle guessed that it was missing here and decided to add it as part of the upgrade